services/wireguard/service/service_unix.go

```diff
@@ -112,7 +107,7 @@ func (manager *Manager) ProvideConfig(publicKey json.RawMessage) (session.Servic
 		}
 	}
 
-	return config, manager.once(destroy), nil
+	return config, destroy, nil
```

Comment:
This seems to be unrelated to PR

Reply:
Well, this is revert of the hack that was added because session destroys never worked. This PR fix session destroys, so these changes can be reverted. 
I can do it in another PR, but why not to include it here? :)

Reply:
Didn't notice the relation. All good 👍